### ISSUE TITLE
Objective coefficients not read correctly from SBML

### ISSUE THREAD

**Issue by matthiaskoenig** [opencobra/cobrapy#464]
The objective coefficients are not read correctly from SBML files. See example attached.

[diauxic_fba.xml.zip](https://github.com/opencobra/cobrapy/files/861442/diauxic_fba.xml.zip)

```
from __future__ import print_function
import pandas as pd

import cobra
print(cobra.__version__)


def cobra_reaction_info(cobra_model):
    """ Creates data frame with bound and objective information.

    :param cobra_model:
    :return: pandas DataFrame
    """
    rids = [r.id for r in cobra_model.reactions]
    df = pd.DataFrame(data=None, index=rids,
                      columns=['lb', 'ub', 'reversibility', 'objective', 'boundary'])
    # FIXME: better filling of DataFrame
    for rid in rids:
        r = cobra_model.reactions.get_by_id(rid)
        df.loc[rid] = [r.lower_bound, r.upper_bound, r.reversibility, r.objective_coefficient, r.boundary]
    return df


sbml_path = "diauxic_fba.xml"
model = cobra.io.read_sbml_model("diauxic_fba.xml")

print('* cobrapy coefficients *')
df = cobra_reaction_info(model)
print(df)

print('* real coefficients *')
import libsbml
doc = libsbml.readSBMLFromFile(sbml_path)

# one model, one objective
m = doc.getModel()
fbc_m = m.getPlugin("fbc")
obj = fbc_m.getObjective(0)
print(obj)
for fobj in obj.getListOfFluxObjectives():
    print(fobj.getReaction(), ' = ', fobj.getCoefficient())

print()
print(model.reactions.v1.objective_coefficient)
print(model.reactions.v2.objective_coefficient)
print(model.reactions.v3.objective_coefficient)
print(model.reactions.v4.objective_coefficient)
```

cobrapy only sets v4 coefficient !!!
```
0.6.0a3
* cobrapy coefficients *
            lb    ub reversibility objective boundary
v1           0  1000         False         0    False
v2           0  1000         False         0    False
v3           0  1000         False         0    False
v4           0  1000         False         1    False
EX_O2        0  1000         False         0     True
EX_Glcxt     0  1000         False         0     True
EX_Ac    -1000  1000          True         0     True
EX_X         0  1000         False         0     True
* real coefficients *
<Objective biomass_max>
v1  =  1.0
v2  =  1.0
v3  =  1.0
v4  =  1.0

0
0
0
1.0
```
This is a major bug, because all simulations from SBML files are wrong with this.

**Comment by hredestig** [comment]
Oopss.. thanks for reporting! my mistake here, fixing..

**Comment by hredestig** [comment]
Should be fixed with #465 
```
(35new-cameo) ➜  cobrapy git:(fix/support-multiple-objective-coef-sbml) ✗ python bug.py
0.6.0a3
* cobrapy coefficients *
            lb    ub reversibility objective boundary
v1           0  1000         False         1    False
v2           0  1000         False         1    False
v3           0  1000         False         1    False
v4           0  1000         False         1    False
EX_O2        0  1000         False         0     True
EX_Glcxt     0  1000         False         0     True
EX_Ac    -1000  1000          True         0     True
EX_X         0  1000         False         0     True
* real coefficients *
<Objective biomass_max>
v1  =  1.0
v2  =  1.0
v3  =  1.0
v4  =  1.0

1.0
1.0
1.0
1.0
```

**Comment by matthiaskoenig** [comment]
Thanks for this.
